### PR TITLE
Changelog update for targetResource changes.

### PR DESCRIPTION
**Pull request by lleonard-msft**
- Updating changelog.  Additional changes expected.

### CODE REVIEW

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[445891c](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4016/commits/445891cc0fe0f6a877c422c56d33a4eb96b23358)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 25 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[445891c](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4016/commits/445891cc0fe0f6a877c422c56d33a4eb96b23358)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 2 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[445891c](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4016/commits/445891cc0fe0f6a877c422c56d33a4eb96b23358)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/changelog.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C20%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902201817132753-4016%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ibTkAhti38ibzywc4rgn%2B62SUSgVrgvEevSN0mfEoXc%3D&st=2019-02-20T18%3A15%3A34Z&se=2019-03-23T18%3A20%3A34Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[6ae612c](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4016/commits/6ae612cbf06ba5e25f82e50d89bd0dd7c9784349)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 25 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[6ae612c](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4016/commits/6ae612cbf06ba5e25f82e50d89bd0dd7c9784349)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 2 sec(s)_

**Comment by lleonard-msft** [comment]
@dkershaw10 - Please review

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[6ae612c](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4016/commits/6ae612cbf06ba5e25f82e50d89bd0dd7c9784349)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/targetresource.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresource.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourcedevice.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourcedevice.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourcedirectory.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourcedirectory.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourcegroup.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourcegroup.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourceother.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourceother.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourcepolicy.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourcepolicy.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourcerole.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourcerole.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourceserviceprincipal.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourceserviceprincipal.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourceuser.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourceuser.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C21%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902211229135682-4016%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=iDAFoDDQtqqYBgt6VdEdvEXyA55Bbmp%2BM%2BXFmeS0PT0%3D&st=2019-02-21T12%3A27%3A48Z&se=2019-03-24T12%3A32%3A48Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api-reference/beta/resources/targetresource.md [hunk 1]
```diff
@@ -1,34 +1,26 @@
 ---
-title: "targetResource resource type"
-description: "Indicates a collection of  target resource types associated with the audit activity. Each target resource type will inherit the properties outlined below from this resource."
+title: "targetResource resource type - Microsoft Graph API"
+description: "Defines the targetResource entity resource of the Microsoft Graph API which supports audit log reporting organization (tenant) activity."
```
Comment (dkershaw10) [inline api-reference/beta/resources/targetresource.md]: Nit: It's a complex type resource

File: api-reference/beta/resources/targetresource.md [hunk 2]
```diff
@@ -1,34 +1,26 @@
...
+author: "lleonard-msft"
 localization_priority: Normal
+ms.prod: "azure-ad"
 ---
 
 # targetResource resource type
-Indicates a collection of  target resource types associated with the audit activity. Each target resource type will inherit the properties outlined below from this resource.
+
+Indicates a collection of  target resource types associated with the audit activity.
```
Comment (dkershaw10) [inline api-reference/beta/resources/targetresource.md]: It's not a collection.  The property on directoryAudit is a collection of targetResource.
Suggest rewording.  Not sure about "indicates" either.
Maybe:  Represent target resource types that are associated with an audit activity.

File: api-reference/beta/resources/targetresource.md [hunk 3]
```diff
@@ -1,34 +1,26 @@
...
-Directory|[targetResourceDirectory](targetresourcedirectory.md]
-Group|[targetResourceGroup](targetresourcegroup.md)
-Policy|[targetResourcePolicy](targetresourcepolicy.md)
-Role|[targetResourceRole](targetresourcerole.md)
-Service Principal|[targetResourceServicePrincipal](targetresourceserviceprincipal.md)
-User|[targetResourceUser](targetresourceuser.md)
-Other|[targetResourceOther](targetresourceother.md)
+|id|String|Indicates the unique ID of the resource.|
+|displayName|String|Indicates the visible name defined for the resource. Typically specified when the resource is created.|
+|type|String|Describes the resource type.  Example values include `Application`, `Group`, `ServicePrincipal`, and `UserPrincipal`.|
```
Comment (dkershaw10) [inline api-reference/beta/resources/targetresource.md]: 1.  Why are these capitalized?
2. Why UserPrincipal?  It is a user principal, but the entity type is called `user`. 
3. Is this the full set?  What about device, domain, etc?

File: api-reference/beta/resources/targetresource.md [hunk 4]
```diff
@@ -1,34 +1,26 @@
...
+|userPrincipalName|String|When **type** is set to `User`, this includes the user name that initiated the action; `null` for other types.|
```
Comment (dkershaw10) [inline api-reference/beta/resources/targetresource.md]: OK - so here type is User, but above UserPrincipal.  Which is it?

File: concepts/changelog.md [hunk 5]
```diff
@@ -23,7 +23,10 @@ For details about known issues with Microsoft Graph APIs, see [Known issues](kno
 
 | **Change type** | **Version**   | **Description**                          |
 | :-------------- | :------------ | :--------------------------------------- |
-|Change |beta and v1.0| Updated the **companyName** property on the [user](/graph/api/resources/user?view=graph-rest-v1.0) resource to be writeable.|
+| Change |beta and v1.0| Updated the **companyName** property on the [user](/graph/api/resources/user?view=graph-rest-v1.0) resource to be writeable.|
+| Change | beta | The [targetResource](/graph/api/resources/targetresource?view=graph-rest-beta) type is now a complex type and reports property changes as a collection of [modifiedProperty](/graph/api/resources/user?view=graph-rest-beta) values. |
+| Deletion | beta | Because the [targetResource](/graph/api/resources/targetresource?view=graph-rest-beta) type is now a complex type, the following types are no longer supported and have been removed: **targetResourceDevice**, **targetResourceDirectory**, **targetResourceGroup**, **targetResourcePolicy**, **targetResourceRole**, **targetResourceServicePrincipal**, **targetResourceUser**, and **targetResourceOther**. |
```
Comment (dkershaw10) [inline concepts/changelog.md]: I think more accurately:
Prior to this change, each target resource type was modeled using its own derived type -  **targetResourceDevice**, **targetResourceDirectory**, **targetResourceGroup**, **targetResourcePolicy**, **targetResourceRole**, **targetResourceServicePrincipal**, **targetResourceUser**, and **targetResourceOther**.  These derived types have now been removed in favor of a flattened model using a single **targetResource** resource type that can represent all these types.

File: concepts/changelog.md [hunk 6]
```diff
@@ -23,7 +23,10 @@ For details about known issues with Microsoft Graph APIs, see [Known issues](kno
 
 | **Change type** | **Version**   | **Description**                          |
 | :-------------- | :------------ | :--------------------------------------- |
-|Change |beta and v1.0| Updated the **companyName** property on the [user](/graph/api/resources/user?view=graph-rest-v1.0) resource to be writeable.|
+| Change |beta and v1.0| Updated the **companyName** property on the [user](/graph/api/resources/user?view=graph-rest-v1.0) resource to be writeable.|
+| Change | beta | The [targetResource](/graph/api/resources/targetresource?view=graph-rest-beta) type is now a complex type and reports property changes as a collection of [modifiedProperty](/graph/api/resources/user?view=graph-rest-beta) values. |
```
Comment (dkershaw10) [inline concepts/changelog.md]: I think this was a complex type before...  Again it's not a collection.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[d3a0acf](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4016/commits/d3a0acf9b1db4de35a0ec66332c26a4c09d69948)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 25 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[d3a0acf](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4016/commits/d3a0acf9b1db4de35a0ec66332c26a4c09d69948)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 2 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[d3a0acf](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4016/commits/d3a0acf9b1db4de35a0ec66332c26a4c09d69948)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/targetresource.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresource.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourcedevice.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourcedevice.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourcedirectory.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourcedirectory.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourcegroup.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourcegroup.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourceother.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourceother.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourcepolicy.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourcepolicy.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourcerole.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourcerole.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourceserviceprincipal.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourceserviceprincipal.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourceuser.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourceuser.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C21%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902212213353643-4016%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=kcZEMw5UJuSaNZ%2B0YkwfmqEQcbBmkac1WGmMCxRXKO0%3D&st=2019-02-21T22%3A12%3A22Z&se=2019-03-24T22%3A17%3A22Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: concepts/changelog.md [hunk 7]
```diff
@@ -23,7 +23,10 @@ For details about known issues with Microsoft Graph APIs, see [Known issues](kno
 
 | **Change type** | **Version**   | **Description**                          |
 | :-------------- | :------------ | :--------------------------------------- |
-|Change |beta and v1.0| Updated the **companyName** property on the [user](/graph/api/resources/user?view=graph-rest-v1.0) resource to be writeable.|
+| Change |beta and v1.0| Updated the **companyName** property on the [user](/graph/api/resources/user?view=graph-rest-v1.0) resource to be writeable.|
+| Change | beta | The [targetResource](/graph/api/resources/targetresource?view=graph-rest-beta) type now represents property changes as [modifiedProperty](/graph/api/resources/user?view=graph-rest-beta) values. |
```
Comment (dkershaw10) [inline concepts/changelog.md]: This type has been updated to include properties that were formerly on the derived type (as part of flattening the model).

Comment (lleonard-msft) [reply concepts/changelog.md]: Fixed.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[29fcf0e](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4016/commits/29fcf0ec5914319d2f7d0b98e1834afd74ff7db9)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 25 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[29fcf0e](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4016/commits/29fcf0ec5914319d2f7d0b98e1834afd74ff7db9)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/microsoftgraph/microsoft-graph-docs, working directory: W:\mjx1-s, target branch: master, target branch commit: 9a7c1cb164f453ac2cd35ee645456c77adf04352, source branch: 98fea34d321931089d41f11e80650c3a/update/target-resource-changelog, source branch commit: .

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C25%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902251919506238-4016%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=k5CXaAWzbXUAbKPmF%2FLwJPAyeOs3hbTYdj4tiKkERH0%3D&st=2019-02-25T19%3A15%3A43Z&se=2019-03-28T19%3A20%3A43Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[49f8791](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4016/commits/49f8791847879d1c3fe9fe989bebca4df8076dbb)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 25 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[49f8791](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4016/commits/49f8791847879d1c3fe9fe989bebca4df8076dbb)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 3 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[49f8791](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4016/commits/49f8791847879d1c3fe9fe989bebca4df8076dbb)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/targetresource.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresource.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourcedevice.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourcedevice.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourcedirectory.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourcedirectory.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourcegroup.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourcegroup.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourceother.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourceother.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourcepolicy.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourcepolicy.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourcerole.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourcerole.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourceserviceprincipal.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourceserviceprincipal.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourceuser.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourceuser.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C25%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902251923423138-4016%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=UYmLAasyBYeaIEbCYX%2FMSeTjGfY132Yfp7ZZQons9Xs%3D&st=2019-02-25T19%3A22%3A11Z&se=2019-03-28T19%3A27%3A11Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by lleonard-msft** [comment]
#sign-off

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[5a670e9](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4016/commits/5a670e9b8e4e675b5d2a69316dd3fd98f10b1d80)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 25 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[5a670e9](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4016/commits/5a670e9b8e4e675b5d2a69316dd3fd98f10b1d80)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 3 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[5a670e9](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4016/commits/5a670e9b8e4e675b5d2a69316dd3fd98f10b1d80)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/targetresource.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresource.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourcedevice.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourcedevice.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourcedirectory.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourcedirectory.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourcegroup.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourcegroup.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourceother.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourceother.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourcepolicy.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourcepolicy.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourcerole.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourcerole.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourceserviceprincipal.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourceserviceprincipal.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourceuser.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourceuser.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C28%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902281701418958-4016%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=4RXQbYgBU1fqFOqoTzLL%2BdMzQyyRecbbX5L0D2E6j5o%3D&st=2019-02-28T17%3A00%3A40Z&se=2019-03-31T17%3A05%3A40Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[f3416fb](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4016/commits/f3416fb3c7c1cfeabee65c6d7aa95f4267bb3b08)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 25 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[f3416fb](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4016/commits/f3416fb3c7c1cfeabee65c6d7aa95f4267bb3b08)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 3 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[f3416fb](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4016/commits/f3416fb3c7c1cfeabee65c6d7aa95f4267bb3b08)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/targetresource.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresource.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourcedevice.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourcedevice.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourcedirectory.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourcedirectory.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourcegroup.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourcegroup.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourceother.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourceother.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourcepolicy.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourcepolicy.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourcerole.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourcerole.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourceserviceprincipal.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourceserviceprincipal.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourceuser.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourceuser.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C28%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902282249051741-4016%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=MvAgZS%2B3pFEm9XyL5QZW4gOyX6ukKeseOxPcdVelylM%3D&st=2019-02-28T22%3A47%3A41Z&se=2019-03-31T22%3A52%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[97e7a92](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4016/commits/97e7a92a1dd586a44d361f90b5d8498324ec51ab)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 25 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[97e7a92](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4016/commits/97e7a92a1dd586a44d361f90b5d8498324ec51ab)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 3 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[97e7a92](https://github.com/lleonard-msft/microsoft-graph-docs/pull/4016/commits/97e7a92a1dd586a44d361f90b5d8498324ec51ab)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/targetresource.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresource.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourcedevice.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourcedevice.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourcedirectory.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourcedirectory.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourcegroup.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourcegroup.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourceother.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourceother.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourcepolicy.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourcepolicy.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourcerole.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourcerole.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourceserviceprincipal.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourceserviceprincipal.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/targetresourceuser.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/api-reference/beta/resources/targetresourceuser.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/target-resource-changelog/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C28%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902282303415420-4016%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=lwZ6QFUG62rNDVhOe5kXWqsHt0qtcAWRPSmLV4YBAqo%3D&st=2019-02-28T23%3A02%3A23Z&se=2019-03-31T23%3A07%3A23Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.
